server/src/main.ts

```diff
@@ -110,7 +110,8 @@ if (!gotSingleInstanceLock) {
     app.quit();
 } else {
     const serverConfig = serverConfigFactory(
-        process.env.USERDATA || app.getPath('userData')
+        process.env.USERDATA || app.getPath('userData'),
+        process.env.TEMPDATA || app.getPath('temp')
```

Comment:
The temp path is generic and not app specific. In the player, I've appended a directory:
`const tmpDir = path.join(app.getPath('temp'), 'lumiplayer')`